pkgs/test/test/runner/browser/code_server.dart

```diff
@@ -35,31 +35,6 @@ class CodeServer {
 
   CodeServer._(this._handler, this.url);
 
-  /// Sets up a handler for the root of the server, "/", that serves a basic
-  /// HTML page with a script tag that will run [dart].
-  void handleDart(String dart) {
```

Comment:
Was this just unused? Curious how you were able to determine that?

Reply:
Yea, it was just unused. In this case, I was curious how it was used, and noticed there were no references. Grepping also showed no uses.

I'm happy to add it back if you decide it has value.